app/celery/letters_pdf_tasks.py

```diff
@@ -173,9 +189,53 @@ def collate_letter_pdfs_to_be_sent():
             )
         current_app.logger.info(f"finished collate-letter-pdfs-to-be-sent processing for postage class {postage}")
 
+    send_letters_volume_email_to_dvla(letters_volume)
     current_app.logger.info("finished collate-letter-pdfs-to-be-sent")
 
 
+def send_letters_volume_email_to_dvla(letters_volume):
```

Comment:
Have you considered using a query to get the volume data?
It would mean `send_letters_volume_email_to_dvla` could be run on it's own or as a task and we can send the email before we start sending the files. The query is efficient and would look something like this: 
```
select count(*), sum(billable_units), postage
from notifications 
where notification_type = 'letter'
and key_type = 'normal'
and notification_status = 'created'
and created_at <= print_run_deadline
group by postage
```

Reply:
Hmmm, that is a sound suggestion.